internal/publish/database/exposure.go

```diff
@@ -154,8 +157,14 @@ func generateExposureQuery(criteria IterateExposuresCriteria) (string, []interfa
 	`
 
 	if len(criteria.IncludeRegions) == 1 {
-		args = append(args, criteria.IncludeRegions)
-		q += fmt.Sprintf(" AND (regions && $%d)", len(args)) // Operation "&&" means "array overlaps / intersects"
+		if criteria.IncludeTravelers {
+			args = append(args, criteria.IncludeRegions)
+			args = append(args, true)
+			q += fmt.Sprintf(" AND ((regions && $%d) OR traveler = $%d)", len(args)-1, len(args)) // Operation "&&" means "array overlaps / intersects"
```

Comment:
I'm pretty confused about what's happening here in this SQL. Could we break this up or add a comment?

Also, I'm hesitant about passing the raw SQL string instead of binding these values. I understand they are lengths that we calculate, but it's still derived from user-supplied input.

Reply:
We're not passing the raw SQL - it's setting values for bind variables and simultaneously building up the list.

Reply:
added comment